### PR TITLE
parrot-ar-drone: make it playable and improve it

### PR DESCRIPTION
**Pull request by ds-hwang**
There are 6 major changes:
 - Use unprefix gamepad API to run on chrome after M32
 - Use chrome.sockets.(udp|tcp) API instead of deprecated chrome.socket API
 - Make it possible to reconnect to Drone after network error.
 - Add more features e.g. flip, emergency
 - Display drone's states
 - Fix many bugs

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Thank you really much Dongseong! This looks :+1: 
I'd like to try it out first with a drone first before merging if that's OK with you.

In the mean time, do you mind bumping app version in the manifest file?

Moreover, as you may already know, we require contributors to sign Google's open source Contributor License Agreement. It's easy to do, just click here to sign as an individual or corporation. See https://github.com/GoogleChrome/chrome-app-samples/blob/2af9f9e82bd762ebe1bfb413356ad596ae0c95eb/CONTRIBUTING.md

File: samples/parrot-ar-drone/lib/API.js [hunk 1]
```diff
@@ -119,51 +133,78 @@ DRONE.API = (function() {
...
+      disconnect(sockets['nav']);
+      disconnect(sockets['vid']);
+      disconnect(sockets['ctrl']);
+    } catch (err) {
+      sockets['at'].socket = null;
+      sockets['nav'].socket = null;
+      sockets['vid'].socket = null;
+      sockets['ctrl'].socket = null;
+    }
+    log("disconnected. push X to reconnect");
```
Comment (beaufortfrancois) [inline samples/parrot-ar-drone/lib/API.js]: Nit: `s/push/press/g`

**Comment by beaufortfrancois** [comment]
I've been playing with the app and a Drone and I have to say there were a lot of disconnections. Is that something you've experienced as well?

**Comment by ds-hwang** [comment]
I didn't have experience more disconnection than before. Sometimes disconnection happens due to UDP socket disconnection but we can reconnect by pressing 'X'

**Comment by ds-hwang** [comment]
@googlebot I'm a chromium committer and signed a CLA via Intel. In addition, I've already contributed on this project before.

**Comment by beaufortfrancois** [comment]
Thank you.

Before merging your PR, can you address the small nit https://github.com/GoogleChrome/chrome-app-samples/pull/372#discussion_r27463857 and bump app version in the `manifest.json` file?

**Comment by ds-hwang** [comment]
@beaufortfrancois , Thank you for reviewing!

I addressed nits and bump version up

**Comment by beaufortfrancois** [comment]
Thank you really much Dongseong! 
I just merged it and updated the Webstore Sample at https://chrome.google.com/webstore/detail/drone-controller-sample/lhdfniaagbjbipjmgfbnlbcmlbcgklkh

**Comment by ds-hwang** [comment]
@beaufortfrancois thank you for reviewing. I'm enabling video. I'll submit PR soon.

**Pull request merged**